integrationTests/testProcessorNode.go

```diff
@@ -999,12 +999,18 @@ func (tpn *TestProcessorNode) createDefaultEconomicsConfig() *config.EconomicsCo
 			},
 		},
 		FeeSettings: config.FeeSettings{
-			MaxGasLimitPerBlock:     maxGasLimitPerBlock,
-			MaxGasLimitPerMetaBlock: maxGasLimitPerBlock,
-			MinGasPrice:             minGasPrice,
-			MinGasLimit:             minGasLimit,
-			GasPerDataByte:          "1",
-			GasPriceModifier:        0.01,
+			GasLimitSettings: []config.GasLimitSetting{
```

Comment:
Should add a new task for adding integration tests for this new split mechanism. This is required in order to not have regressions on the process.

Reply:
DOne. https://elrondnetwork.atlassian.net/browse/EN-10917